ci/Jenkinsfile.qt-build

```
@@ -0,0 +1,224 @@
+        }
+
+        stage('AppImage Builder') {
+            steps {
+                script {
+                    echo "Building AppImage Builder image..."
+                    sh """
+                        cd ${QT_BUILD_DIR}/appimage-builder
+                        docker build -t ${DOCKER_REGISTRY}/appimage-builder:noble .
+                    """
+                }
+            }
+        }
+
+        stage('Qt Dev Base') {
+            steps {
+                script {
+                    echo "Building Qt Dev Base image..."
+                    sh """
+                        cd ${QT_BUILD_DIR}/base
+                        docker build \
+                            --build-arg TARGETARCH=amd64 \
+                            -t ${DOCKER_REGISTRY}/qt-dev-base:noble-17-35-27.2.12479018 .
+                    """
+                }
+            }
+        }
+
+        stage('Qt Android') {
+            when {
+                expression { params.BUILD_QT_FROM_SOURCE == true }
+            }
+            agent {
+                docker {
+                    label 'linux && qt-builder'
+                    image "${DOCKER_REGISTRY}/qt-dev-base:noble-17-35-27.2.12479018"
+                    args "-u root -v ${WORKSPACE}/qt_export:/root/export -v ${WORKSPACE}/ci/docker/qt-build/scripts:/root/scripts"
+                }
+            }
+            steps {
+                script {
+                    echo "Building Qt ${params.QT_VERSION} for Android..."
+                    sh """
+                        chmod +x /root/scripts/build_qt_android.sh /root/scripts/build_qt_desktop.sh
+                        cd /root
+                        /root/scripts/build_qt_android.sh
+                    """
+                }
+            }
+        }
+
+        stage('Build Qt CI Image') {
+            steps {
+                script {
+                    echo "Building final Qt Docker image..."
+                    sh """
+                        cd ${QT_BUILD_DIR}/qt
+                        cp -r ${WORKSPACE}/qt_export .
+                        docker build \
+                            --build-arg QTVER=${params.QT_VERSION} \
+                            -t ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG} .
+                    """
+                }
+            }
+        }
+
+        stage('Build Enhanced Runtime Image') {
+            steps {
+                script {
+                    echo "Building enhanced runtime image with Nim and all tooling..."
+                    sh """
+                        cd ${QT_BUILD_DIR}/nim-runtime
+                        docker build \
+                            --build-arg GOLANG_VERSION=1.23.10 \
+                            --build-arg NIM_VERSION=2.0.12 \
+                            --build-arg NIX_VERSION=2.24.11 \
+                            -t ${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG}-ci .
+                    """
+                }
+            }
```

How is this image "enhanced"? I don't get it.
```suggestion
            steps { script {
                dir("${QT_BUILD_DIR}/nim-runtime") {
                    ciImage = docker.build(
                        "${DOCKER_REGISTRY}/${DOCKER_IMAGE_BASE}:${params.DOCKER_TAG}-ci",
                        "--build-arg GOLANG_VERSION=1.23.10 " +
                        "--build-arg NIM_VERSION=2.0.12 " +
                        "--build-arg NIX_VERSION=2.24.11 " +
                        "."
                    )
                }
            } }
```